src/atc_pyspark.py

```diff
@@ -0,0 +1,33 @@
+from pyspark.sql import SparkSession
+from pyspark.sql.types import *
+import pyspark.sql.functions as F
+from pyspark.sql.types import (
+    ArrayType, LongType, StringType, StructField, StructType)
+
+spark = SparkSession.builder.getOrCreate()
+df = spark.read.format('xml').options(rowTag='drug').load('s3n://rxminer/drugbank/drugbank.xml')
+df.createOrReplaceTempView("input_table")
+df.show(10)
+df1 = spark.sql("""
+SELECT atc-codes.atc-code AS atc_code
+FROM input_table
+""")
+
+# df = df.select('name','products', 'atc-codes')
```

Comment:
Would be good to get rid of commented code for later commits; but if you do need these comments for later use, you can add another comment above stating that this might be needed later